advance_ai_agents/candidate analyser/main.py

```
@@ -0,0 +1,202 @@
+                        if hasattr(chunk, "content") and isinstance(chunk.content, str):
+                            full_response += chunk.content
+                            placeholder.markdown(full_response, unsafe_allow_html=True)
+
+                    # Extract Score from response (e.g., "85/100")
+                    score = 0
+                    match = re.search(r"(\d{1,3})/100", full_response)
+                    if match:
+                        score = int(match.group(1))
+                        st.success(f"Candidate Score: {score}/100")
+
+            except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)